results/pkg/api/server/server_test.go

```diff
@@ -406,3 +495,24 @@ func TaskRunResult(tr *pb.TaskRun) *pb.Result {
 		}},
 	}
 }
+
+func PipelineRunResult(pr *pb.PipelineRun) *pb.Result {
+	return &pb.Result{
+		Executions: []*pb.Execution{{
+			Execution: &pb.Execution_PipelineRun{pr},
+		}},
+	}
+}
+
+func Result(tr *pb.TaskRun, pr *pb.PipelineRun) *pb.Result {
+	var executions []*pb.Execution = make([]*pb.Execution, 0, 2)
+	if tr != nil {
+		executions = append(executions, &pb.Execution{Execution: &pb.Execution_TaskRun{tr}})
+	}
+	if pr != nil {
+		executions = append(executions, &pb.Execution{Execution: &pb.Execution_PipelineRun{pr}})
+	}
+	return &pb.Result{
+		Executions: executions,
+	}
+}
```

Comment:
Making this variadic would let you pass in any combination of Task/PipelineRuns.

```suggestion
func Result(in ...proto.Message) *pb.Result {
	executions := make([]*pb.Execution, 0, len(in))
	for _, m := range in {
		switch x := m.(type) {
		case pb.TaskRun:
			executions = append(executions, &pb.Execution{Execution: &pb.Execution_TaskRun{x}})
		case pb.PipelineRun:
			executions = append(executions, &pb.Execution{Execution: &pb.Execution_PipelineRun{x}})
		default:
			panic("unknown message: %v", m)
		}
	}	
	return &pb.Result{
		Executions: executions,
	}
}
```